assets/css/common.css

```diff
@@ -68,3 +68,16 @@ a.white-link:hover {
 
 /* align labels when material-icons are nearby */
 i.material-icons, i.material-icons-outlined { vertical-align: middle; }
+
+/* accessibility - used to optimize the page for screenreaders */
+.sr-only {
```

Comment:
add reference to [where this came from](https://a11y-guidelines.orange.com/web_EN/exemples/masquage/index.html)